INSTALL.md

```diff
@@ -31,7 +31,7 @@ To setup the Elektra low-level version of LCDproc follow these steps:
   `/sw/lcdproc/lcdd/#0/current/server/driverpath` to `/usr/local/lib/lcdproc` (or whatever directory
   you installed the drivers to).
   ```sh
-  kdb set '/sw/lcdproc/lcdd/#0/current/server/driverpath' /usr/local/lib/lcdproc
+  kdb set '/sw/lcdproc/lcdd/#0/current/server/driverpath' /usr/local/lib/lcdproc/
```

Comment:
Is this necessary? Why not have this compiled-in as part of the specification?

Reply:
I always needed to set this so the path is working.
I am not sure how you mean that "as part of the specification".

Reply:
I created #6 for further discussions.